src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaveForBarnTaskController.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.familie.ef.iverksett.oppgave
+
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.prosessering.domene.Task
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.Properties
+
+@RestController
+@RequestMapping(
+        path = ["/api/iverksett/task/barnfylleraar"],
+        produces = [MediaType.APPLICATION_JSON_VALUE]
+)
+@ProtectedWithClaims(issuer = "azuread")
+class OpprettOppgaveForBarnTaskController(
+        private val oppgaveForBarnTask: OpprettOppgaveForBarnTask
+) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    @PostMapping("/oppgaverforbarn")
+    fun opprettTask(@RequestBody oppgaverForBarn: OppgaverForBarn): ResponseEntity<Unit> {
+        oppgaverForBarn.oppgaverForBarn.forEach {
```

Comment:
Dette burde nog flyttes til servicen, og få en `@Transactional` 
Samtidig så burde taskene opprettes i databasen, dvs `taskRepository.save()` og ikke kalle på task-servicen direkte, då får vi ikke håndteringen av tasker som taskbiblioteker gir oss.
Og får du feilhåndtering for hver oppgave som skal opprettes